search-service/src/query_representation/initial/mod.rs

```diff
@@ -286,4 +303,36 @@ mod private_tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_terminal_expression_to_simple_with_attr_as_value() -> Result<(), Error> {
```

Comment:
Bom caso, mas nós pensamos em como isso vai ser feito no front-end? 

Não consigo imaginar uma interface para isso. Tínhamos pensando em nome do atributo + operação + um caixa onde o usuário vai poder escrever o valor do atributo

Reply:
Pelo que eu entendi (@Caio-Dantas pode dizer melhor), mas enquanto o usuário digita os valores a gente pode ir checando se é um atributo válido. Se não for tudo bem, aí a gente considera que é um valor qualquer. Na prática acho que vai ser só implementar igual a caixa do atributo, mas permitindo que ele coloque uma string q não está na lista de atributos válidos.